README.md

```diff
@@ -56,146 +85,82 @@ OBSERVER_LOG_OUTPUTTYPE=stderr
-
-2. Import [this Grafana dashboard](https://github.com/insolar/observer/blob/master/scripts/monitor/grafana/dashboards/observer.json) into Grafana. 
- 
-   If you need to, [read how to import a dashboard]( https://grafana.com/docs/grafana/latest/reference/export_import/).
+Run the Observer and wait for a while for it to sync with the trusted HMN: `./bin/observer --config .artifacts/observer.yaml`
 
+Tip: To run the Observer, you need to use the `observer.yaml` configuration file. The file should be in the `.artifacts` directory.
 
-## API service
+### Deploy the Observer API
 
-To run API you should provide config file `observerapi.yaml`
-in the current working directory or in `.artifacts` directory.
+#### Configure the Observer API 
+All options in observerapi.yaml config can be overridden with environment variables using OBSERVERAPI prefix and _ as delimiter, for example: OBSERVERAPI_DB_URL=..., OBSERVERAPI_LISTEN=...
 
-Run `./bin/api -- config .artifacts/observerapi.yaml`
+**WARNING**: overriding via ENV variables works only with the configuration file in place with the default number of parameters.
 
-### Configure the API
-
-All parameters in `observer.yaml` can be overridden via environment variables that start with `OBSERVER` and use `_` as a separator. For example: `OBSERVER_DB_URL=...`, `OBSERVER_REPLICATOR_LISTEN=...`
-
-**WARNING:** overriding via ENV variables doesn't work without a configuration file in place.
-
-### Configuration parameters
+##### Configuration parameters
 
 API endpoint:
-`OBSERVERAPI_LISTEN=127.0.0.1:5678`
-or
-`OBSERVERAPI_LISTEN=:5678`
+`OBSERVERAPI_LISTEN=127.0.0.1:5678 or OBSERVERAPI_LISTEN=:5678`
 
 Database connection:
 `OBSERVERAPI_DB_URL=postgres://user:password@host/db_name?sslmode=disable`
 
-Max number of connections to the database:
+Maximum number of connections to the database: 
 `OBSERVERAPI_DB_POOLSIZE=20`
 
-Log parameters:
+Log params:
 ```
 OBSERVERAPI_LOG_LEVEL=info
 OBSERVERAPI_LOG_FORMAT=text
 OBSERVERAPI_LOG_OUTPUTTYPE=stderr
 OBSERVERAPI_LOG_BUFFER=0
 ```
 
-Coin statistic API:
-You need to choose an origin of coin statistic.
+#### Run the Observer API
 
-```
-PriceOrigin = const|binance|coin_market_cap
-const - By default it's taken from values from config (somekind of hardcode)
-binance - get data gathered by binance collector
-coin_market_cap - get data gathered by cmc collector
-```
+Run the Observer API: `./bin/api`.
 
-## Stats collector
-Command calculates, gathers and saves statistics, add to cron for 1/min execution.
-Uses replicator's config (see above).
+Note: To run the Observer, you need to use the `observerapi.yaml` configuration file. The file should be in the `.artifacts` directory.
 
-## Binance collector
-Binance gathers info about the exchange rate of the INS/USD pair (could be calculated like a XNS/USD)
+These endpoints are restricted from Observer users:
```

Comment:
I suppose, we shouldn't write about restricted endpoints here, if we want to deploy the particular spec to the special address.

Reply:
Changed to "Observer users can access the following endpoints:" with a "TBD" in the list of points.